.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+# Text is UTF-8
+charset = utf-8
+# Unix-style newlines
+end_of_line = lf
+# Newline ending every file
+insert_final_newline = true
+# Soft tabs
+indent_style = space
+# Two-space indentation
```

Comment:
This comment seems to be wrong :-D The actual value is correct though.

Reply:
Whoops! Fixed.